parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestParquetFileWriter.java

```diff
@@ -425,6 +436,27 @@ public void testConvertToThriftStatistics() throws Exception {
   }
 
   @Test
+  public void testStatisticsGettingIgnoredFor160Version() throws Exception {
+    long[] longArray = new long[] {39L, 99L, 12L, 1000L, 65L, 542L, 2533461316L, -253346131996L, Long.MAX_VALUE, Long.MIN_VALUE};
+    LongStatistics parquetMRstats = new LongStatistics();
```

Comment:
i think there's a method on stats like isEmpty() that has to be inspected before using the object IIRC, let me look....